### PR TITLE
Css frameworks rebuild

### PR DESCRIPTION
**Pull request by Phmik**
- Removed styles
- Added more BS
- Made it more responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                            </div>
+                          </div>
+                        </nav>  
+                    </div>
+                </div>
+            </div>
+        </header>
+        <main>
+            <section class="user-posts">
+                <div class="container m-0">
```
Comment (Fermain) [inline index.html]:
```suggestion
                <div class="container m-0">
```

This sticks out a bit. One of the purposes of a container is to use margin auto to horizontally align the page. If you don't want this you should override the whole container class instead so it's consistent.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            <div class="container">
+                <div class="row justify-content-center">
+                    <div class="col">
+                        <img src="assets/images/logo.png" alt="image of the phillers logo">
+                    </div>
+                    <hr>
+                    <div class="row mb-4">
+                      <div class="col-12">
+                        <nav class="nav align-items-center">
+                            <ul class="list-inline col-12 col-md-8 align-self-start d-flex justify-content-start">
```
Comment (Fermain) [inline login.html]: The parent of a col should be a row. It's probably best to separate out a new div here:

```suggestion
<div class="col-12 col-md-8">
  <ul class="list-inline row align-self-start justify-content-start">
```

**Review by Fermain — CHANGES_REQUESTED** [review]
This is a big improvement from the first submission with only a couple of minor comments.

File: index.html (hunk 1 above)
Comment (Phmik) [reply index.html]: I kinda wanted it to stick to the wrappers max width

Comment (Fermain) [reply index.html]: Then you could use `.container-fluid`

File: login.html (hunk 2 above)
Comment (Phmik) [reply login.html]: I'll add that when I get back on my computer. My workplace doesn't really give me opportunity to edit codes :D

File: index.html (hunk 1 above)
Comment (Phmik) [reply index.html]: Cool. I'll check em both out and see what fits the best! Thanks man

**Pull request merged**